crates/katana/rpc/rpc/src/dev.rs

```diff
@@ -0,0 +1,55 @@
+use std::sync::Arc;
+
+use jsonrpsee::core::{async_trait, Error};
+use katana_core::sequencer::KatanaSequencer;
+use katana_primitives::FieldElement;
+use katana_rpc_api::dev::DevApiServer;
+use katana_rpc_types::error::katana::KatanaApiError;
+
+pub struct DevApiImpl {
+    sequencer: Arc<KatanaSequencer>,
+}
+
+impl DevApiImpl {
+    pub fn new(sequencer: Arc<KatanaSequencer>) -> Self {
+        Self { sequencer }
+    }
+}
+
+#[async_trait]
+impl DevApiServer for DevApiImpl {
+    async fn generate_block(&self) -> Result<(), Error> {
+        self.sequencer.block_producer().force_mine();
+        Ok(())
+    }
+
+    async fn next_block_timestamp(&self) -> Result<u64, Error> {
+        // Ok(self.sequencer.backend().env.read().block.block_timestamp.0)
+        unimplemented!()
```

Comment:
```suggestion
        Ok(())
```

This already exist before, but while we're at it can we just remove the panic here.